rs/bitcoin/kyt/templates/dashboard.html

```diff
@@ -0,0 +1,113 @@
+{% macro btc_tx_link(txid) -%}
+    {% match btc_network %}
+        {%- when BtcNetwork::Mainnet -%}
+            <a href="https://live.blockcypher.com/btc/tx/{{txid}}"><code>{{txid}}</code></a>
+        {%- when BtcNetwork::Testnet -%}
+            <a href="https://live.blockcypher.com/btc-testnet/tx/{{txid}}"><code>{{txid}}</code></a>
+    {% endmatch %}
+{%- endmacro %}
+
+{% macro btc_address(addr) -%}
+    {% match addr %}
+        {%- when Some(addr) -%}
+            {% if crate::blocklist_contains(addr) %}
+                <code style="color: red">{{addr}}</code>
+            {% else %}
+                <code style="color: green">{{addr}}</code>
+            {% endif %}
+        {%- when None -%}
+        <code>N/A</code>
+    {% endmatch %}
+{%- endmacro %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>KYT Canister Dashboard for Bitcoin ({{ btc_network }})</title>
+    <style>
+        table {
+            border: solid;
+            text-align: left;
+            width: 100%;
+            border-width: thin;
+        }
+
+        h3 {
+            font-variant: small-caps;
+            margin-top: 30px;
+            margin-bottom: 5px;
+        }
+
+        table table {
+            font-size: small;
+        }
+
+        .background {
+            margin: 0;
+            padding: 0;
+        }
+
+        .content {
+            max-width: 100vw;
+            width: fit-content;
+            margin: 0 auto;
+        }
+
+        tbody tr:nth-child(odd) {
+            background-color: #eeeeee;
+        }
+
+        tbody tr:nth-child(even) {
+            background-color: #ffffff;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="background">
+        <div class="content">
+            <h2>KYT Canister Dashboard for Bitcoin ({{ btc_network }})</h2>
+            <h3>Metadata</h3>
+            <table>
+                <tbody>
+                    <tr id="outcall-capacity">
+                        <th>Outcall Capacity</th>
+                        <td><code>{{ outcall_capacity }}</code></td>
+                    </tr>
+                </tbody>
+            </table>
+            <h3 id="fetch-tx-status">Fetch Transaction Status</h3>
+            <table>
+                <thead>
+                    <th>Txid</th>
+                    <th>Created</th>
+                    <th>Status</th>
+                </thead>
+                <tbody>
+                    {% for (txid, timestamp, status) in fetch_tx_status %}
+                    <tr>
+                        <td>{% call btc_tx_link(txid) %}</td>
+                        <td><code>{{ timestamp|timestamp_to_datetime }}</code></td>
+                        <td>
+                            <table><thead><th>{{ status.to_str() }}</th></thead>
```

Comment:
nit: Why not implement Display on `Status` so that you can just have
```suggestion
                            <table><thead><th>{{ status }}</th></thead>
```

Reply:
Fixed.